### DISCUSSION TITLE
Date picker, place hours-minutes bar right to date.

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#3069] [Ideas]
Now if user wants to select time he has to press this button:
![Снимок экрана (118)_LI](https://user-images.githubusercontent.com/78722918/219852387-814d9373-b257-4046-b32d-b2721931ae63.jpg)

and then this menu will appear:
![Снимок экрана (119)](https://user-images.githubusercontent.com/78722918/219852448-5b631845-afbb-48f0-8826-1b71fb976c02.png)

But our users find it inconvenient. Instead they would like this bar to be opened next to main date picker, like placing it right side

**Comment by SevenOutman** [comment]
This could be achieve by using two separate DatePickers side-by-side. One for selecting date and the other for selecting time.

**Reply by abdullaev388** [reply]
yes but i need it to be in one datepicker

**Reply by SevenOutman** [reply]
Let me confirm this...Are you suggesting that the calendar widget for selecting a date and the scroll widget for selecting a time should be displayed side-by-side?

```
----------------------------
|            |             |
|            |             |
|  Calendar  | Hour/minute |
|            |    scroll   |
|            |             |
----------------------------
```

**Reply by abdullaev388** [reply]
Yeah, that would perfect

**Reply by SevenOutman** [reply]
That makes sense. I would discuss with our design team.

**Reply by abdullaev388** [reply]
any news?

**Reply by SevenOutman** [reply]
We're still discussing on it

**Reply by abdullaev388** [reply]
soo

**Reply by SevenOutman** [reply]
We've got some draft designs but not yet have bandwidth to focus on this topic. Please stay tuned.
